admin/helpers/school-home-feature-eligibility-presenter.js

```diff
@@ -185,14 +198,37 @@ schoolHomeFeatureEligibilityPresenter.isHdfPageAccessible = (currentDate, checkW
 }
 
 /**
- * Determine if results page is accessible
+ * Determine if results feature is accessible
+ * @param currentDate
+ * @param checkWindowData
+ * @returns {Boolean}
+ */
+schoolHomeFeatureEligibilityPresenter.isResultsFeatureAccessible = (currentDate, checkWindowData) => {
```

Comment:
Not sure about the word feature in the method name.  Is this the same as Results Page or something different?

Reply:
@jon-shipley this is moved to a dedicated service along with the other 2 methods.
Essentially this determines access to the feature from school home level where the other two methods decide whether unavailable scores page should be displayed instead of the actual results list when the user is browsing from school home page.